app/models/workers_scaling/experiment_resources_interface.rb

```diff
@@ -158,6 +141,38 @@ def get_worker_record_by_sm_uuid(sm_uuid)
     private
 
     ##
+    # Parses requested amount of workers to schedule to real amount.
```

Comment:
@mwrona Luźna uwaga: wydaje mi się, że czasami można pisać bardziej zrozumiałe komentarze, analizując to zdanie wychodzi na to, że parsujemy _(dlaczego parsujemy, skoro to jest liczba w formacie int?)_ żądaną liczbę workerów do schedulowania _(tutaj ok)_ do rzeczywistej ilości _(rozumiem, że w domyśle "of workers", ale "real" nie do końca musi być zrozumiałe dla wszystkich)_.

Nie lepiej np.:
```
Uses +requested_amount+ number of workers intended to schedule, to compute number of workers applicable in provided +infrastructure_configuration+.
```

Coś takiego, ale nie wiem czy do końca zrozumiałem Twoje intencje w tym komentarzu. W każdym razie - żeby komentarz po prostu nie przepisywał nazwy metody w dłuższy sposób wraz z parametrami (nie mówię, że ten akurat taki był).